cfme/fixtures/cli.py

```diff
@@ -0,0 +1,90 @@
+from utils.version import get_stream
+from cfme.test_framework.sprout.client import SproutClient
+from utils.appliance import current_appliance
+from utils.conf import cfme_data, credentials
+from utils.log import logger
+import pytest
+from wait_for import wait_for
+from cfme.test_framework.sprout.client import SproutException
+
+
+@pytest.yield_fixture(scope="function")
+def dedicated_db_appliance(app_creds):
+    if current_appliance.version.vstring > '5.7':
+        sp = SproutClient.from_config()
+        version = current_appliance.version.vstring
+        stream = get_stream(current_appliance.version)
+        apps, pool_id = sp.provision_appliances(
+            count=1, preconfigured=False, version=version, stream=stream)
+        pwd = app_creds['password']
+        client = apps[0].ssh_client
+        channel = client.invoke_shell()
+        stdin = channel.makefile('wb')
+        stdin.write("ap \n 8 \n 1 \n 1 \n 1 \n y \n {} \n {} \n \n".format(pwd, pwd))
```

Comment:
I agree we should have a test for that, but I think we should add that to the appliance_console PR I have opened rather than this cli one. I mean atm this is just a go between fixture which I hope to replace with a new cli method once we get some more feature parity.